evennia/contrib/rpg/traits/traits.py

```diff
@@ -201,16 +202,17 @@ class Object(DefaultObject):
                                   current    value
                                              = current
                                              + mod
+                                             * mult
```

Comment:
This looks wrong since it lacks the parenthesis; it looks like mod * mult. Better reformat it to use the parenthesis.